### ISSUE TITLE
Carefully consider flows_raw sorting key

### ISSUE THREAD

**Issue by leoluk** [leoluk/NetMeta#30]
- Optimize for main dasboard queries.
- Low-cardinality values first for easy filtering (verify that this does what we think it does).
- Sequence Num for sampling, but how? It's as high-cardinality as it gets.
- Play with secondary idexes - are they more efficient than a carefully-considered sorting key?

**Comment by UnamedRus** [comment]
> Play with secondary idexes - are they more efficient than a carefully-considered sorting key?

No, they ~only make sense if you have some data locality for column used in skip index, (so there is some correlation between that column and table ORDER BY key)

> Sequence Num for sampling, but how? It's as high-cardinality as it gets.

Sampling only make sense if you have good distribution of data across whole number range (min-max)

https://kb.altinity.com/altinity-kb-queries-and-syntax/altinity-kb-sample-by/
